resources/sass/app.scss

```diff
@@ -61,19 +56,46 @@ a {
     text-decoration: none;
 }
 
+.store header{
+    flex-direction: row;
+    @media (max-width: $width-breakpoint-tablet) {
+        flex-direction: column-reverse;
+    }
+
+    .auth-widget {
+        @media (max-width: $width-breakpoint-tablet) {
+            margin-bottom: $dimension-layout-xsmall;
+        }
+    }
+}
+
+.website {
+    header {
+        .wikidata-logo {
+            @media (max-width: $width-breakpoint-tablet) {
+                margin-bottom: $dimension-layout-small;
+            }
+        }
+
+        @media (max-width: $width-breakpoint-tablet) {
+            flex-direction: column;
+        }
+    }
+    h1 {
+        @media (max-width: $width-breakpoint-tablet) {
+            margin: $dimension-layout-xxsmall 0;
+        }
+    }
+}
```

Comment:
Instead of putting this in the global sass file, I think it would be best if we put it where it is actually used, In the Layout.vue file